p2p/host/basic/basic_host_test.go

```diff
@@ -605,6 +679,14 @@ func waitForAddrChangeEvent(ctx context.Context, sub event.Subscription, t *test
 	}
 }
 
+type sortedMultiaddrs []ma.Multiaddr
+
+func (sma sortedMultiaddrs) Len() int      { return len(sma) }
+func (sma sortedMultiaddrs) Swap(i, j int) { sma[i], sma[j] = sma[j], sma[i] }
+func (sma sortedMultiaddrs) Less(i, j int) bool {
+	return bytes.Compare(sma[i].Bytes(), sma[j].Bytes()) == 1
```

Comment:
Shouldn’t this be == -1?

Reply:
Completely got rid of this code as we don't really need it.